spec-0008/index.md

```diff
@@ -0,0 +1,227 @@
+- "Juanita Gomez <juanitagomezr2112@gmail.com>"
+- "Seth Larson <sethmichaellarson@gmail.com>"
+- "Lars Grüter <lagru@mailbox.org>"
+  discussion: https://discuss.scientific-python.org/t/spec-8-supply-chain-security
+  endorsed-by:
+
+---
+
+## Description
+
+<!--
+Briefly and clearly describe the recommendation.
+https://github.com/scientific-python/summit-2024/issues/9
+-->
+
+Securing the Open Source supply chain (OSSC) is becoming a more relevant concern in recent years, with examples of sophisticated attacks against the ecosystem (e.g., the 2024 [`xz` utils backdoor](https://en.wikipedia.org/wiki/XZ_Utils_backdoor)) and [malware attacks on PyPI](https://blog.pypi.org/posts/2024-04-10-domain-abuse/) highlighting the need for supply chain security to be taken seriously.
+The Python Software Foundation (PSF) is also taking the importance of the OSSC seriously, as demonstrated by the [creation of the PSF Security Developer in Residence position in 2023](https://pyfound.blogspot.com/2023/06/announcing-our-new-security-developer.html).
+
+With the creation of the [Supply-chain Levels for Software Artifacts (SLSA) framework](https://slsa.dev/) and [OpenID Connect (OIDC)](https://openid.net/developers/how-connect-works/) standard being more widely adopted there are now high level developer tools, maintained by professional security teams, that are available with clear recommendations on how to use them.
+
+This SPEC outlines the requirements for practically adopting these security tools and recommendations with a focus on securely publishing release artifacts.
+Securely building release artifacts will be covered in a later SPEC. This set of recommendations complements the recommendations from [SPEC 6 — Keys to the Castle](https://github.com/scientific-python/specs/blob/main/spec-0006/index.md).
+
+This SPEC is written with GitHub in mind, note these recommendations still apply with other services such as GitLab and we refer project to these services' documentations and guidelines.
+
+## Implementation
+
+With a focus on securing the release artifact distribution process, the following processes and standards should be adopted.
+
+### Document the release process
+
+The release process should be clearly and fully documented in the developer documentation and describe each step to make a release and the permissions required to do so.
+
+### Hardening workflow environment permissions
+
+Workflows that publish release artifacts should have _run triggers_ that require intentional actions by maintainers (e.g., `on workflow_dispatch` in GitHub Actions) and require multiple maintainers to approve the workflow to run (c.f. "Use GitHub Actions environments" section below).
+This is to safeguard the project from any one maintainer having the ability to commit to the default branch and make a release directly.
+
+It is also strongly recommended that the repository requires signed commits so that all releases have a verified commit to which they correspond.
```

Comment:
Add a link to more information on what this is, find out if you are already doing it, how to set it up, etc

Reply:
Is this enough?

```suggestion
It is also strongly recommended that the repository requires [signed commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) so that all releases have a verified commit to which they correspond.
```